gcc/config/riscv/riscv.opt

```diff
@@ -133,10 +133,10 @@ Take advantage of linker relaxations to reduce the number of instructions
 required to materialize symbol addresses.
 
 mzce-sext
-Target Bool Var(riscv_mzce_sext) Init(0)
+Target RejectNegative Mask(SEXT)
 
 mzce-zext
-Target Bool Var(riscv_mzce_zext) Init(0)
+Target RejectNegative Mask(ZEXT)
 
 mzce-cmul
```

Comment:
Sense there no c.mul will occur in gcc patch, I think these option is not  nesscessary, but you can just pass it to binutils use in as part.